x-pack/plugins/integration_assistant/server/integration_builder/data_stream.ts

```diff
@@ -31,6 +33,8 @@ export function createDataStream(
       data_stream_description: description,
       package_name: packageName,
       data_stream_name: dataStreamName,
+      multiline_ndjson_toggle_display: useMultilineNDJSON,
+      multiline_ndjson_toggle_default: useMultilineNDJSON,
```

Comment:
The template does not know that they evaluate to the same thing though

Reply:
Do we want to have this flag to be `true` by default in the manifest. Since this is not visible if the toggle is false any way. So we can change it to `default: true`

Reply:
Sounds good, I'll make this so